src/__tests__/addremove.test.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * @jest-environment jsdom
+ */
+
```

Comment:
- [ ] Nice job so far, However, it seems that you have forgotten to test the function for editing the task description, the function for updating an item's 'completed' status, and the "Clear all completed" function as mentioned in the project requirements below 👇 
![image](https://user-images.githubusercontent.com/55465133/135545736-d2ab2031-81e5-48cf-9480-1b1912c6c1f7.png)

For now, I can not see anywhere you have expected a given object's completed attribute to change to `true` using the `expect` function, the same for all the above functions.
Kindly let's add tests for these functions as well 😉  and remember, tests are considered when you assert with the `expect` function 👍

Reply:
Hi @chaw-bot, the only file available for now in the `__tests__` folder is `addremove.test.js`, but I couldn't find in it the tests for the function of editing the task description or the `clear all completed` function.

Reply:
Oh sorry for that, I understand, you can try now creating a new pull request with the updated branch and request for another code review 👍